src/components/Users/UserService.tsx

```diff
@@ -0,0 +1,37 @@
+import HttpService from '../../services/HttpService';
+
+import {User, RawUser} from './UserTypes';
+
+//===---==--=-=--==---===----===---==--=-=--==---===----//
```

Comment:
I'm trying to develop a pattern where the UserService is available both on the front-end and back-end, but the UserController is only a server file.

Reply:
And that would be *Service and *Controller, not just user